templates/conference/homepage/_sponsors_cms.html

```diff
@@ -0,0 +1,52 @@
+<style type="text/css">
+.sponsor { margin: 1em; }
+</style>
+{% load cms_tags static %}
+
+<script>
+fetch('/sponsors')
```

Comment:
Making this dynamic will create SEO problems for the sponsors and also hinder converting the website to a static version after the conference. I think it would be better to use a standard Django servers side template for this.

Reply:
Good point - updated the MR, it's cleaner this way, too.